### PR TITLE
Inovelli 2-Channel Smart Plug: fixes build error

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy [hunk 1]
```diff
@@ -83,7 +83,7 @@ def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null)
 			def allOff = true
 			childDevices.each {
 				n ->
-					if (n.currentValue("switch")? != "off") allOff = false
+					if (n.currentState("switch") != null && n.currentState("switch").value != "off") allOff = false
```
Comment (tpmanley) [inline devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy]: Any reason that `!= null` is included here but not in the other place?

Comment (greens) [reply devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy]: ...coulda sworn i had it...

Comment (dkirker) [inline devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy]: Wouldn't `n.currentState("switch")?.value != "off"` also work?

Comment (greens) [reply devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy]: Probably.

Comment (dkirker) [reply devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy]: I also seem to recall a conversation with @tpmanley on another PR I had created about the original statement should work even if current value returns null.

**Review by dkirker — APPROVED** [review]

**Pull request merged**